WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -249,7 +249,7 @@ struct AddEditCoupon: View {
                         .padding(.bottom, Constants.verticalSpacing)
 
                         Button {
-                            // This should be replaced with `@FocusState` when we drop support for iOS 14
+                            // TODO: This should be replaced with `@FocusState` when we drop support for iOS 14
```

Comment:
This is an extra change, but not related to the content of this PR 👀

Reply:
Opened an issue here: https://github.com/woocommerce/woocommerce-ios/issues/13362